survey-runner-database/terraform.tfvars.example

```diff
@@ -5,5 +5,6 @@ database_cidrs=["10.30.20.96/28","10.30.20.112/28"]
 
 multi_az = false
 backup_retention_period = 0
+database_allocated_storage = 10
```

Comment:
gp2 minimum storage is 100 so i guess it defaults higher but its good to be explicit :)